src/Core/Engine/Systems/LightManager.cpp

```
@@ -1,23 +1,65 @@
 #include "tracy/Tracy.hpp"
 #include "tracy/TracyOpenGL.hpp"
 
-/**
- * @brief Removes the light with the given identifier from the manager.
- *
- * Does nothing if no light exists for the provided `id`.
- *
- * @param id Identifier of the light to remove.
- */
-void LightManager::RemoveLight(LightID id)
+DirectionLightGPU LightManager::ConvertLightToData(const DirectionalLight& light)
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)